core/src/main/java/com/amplitude/core/Storage.kt

```diff
@@ -20,7 +20,9 @@ interface Storage {
 
     suspend fun writeEvent(event: BaseEvent)
 
-    suspend fun write(key: Constants, value: String)
+    fun write(key: Constants, value: String)
```

Comment:
Why remove suspend? I feel we keep it to make the rest of the interface. wdyt?

Reply:
@justin-fiedler 
I don't know - maybe https://discuss.kotlinlang.org/t/why-not-make-every-function-suspend-by-default/25224 can help
I've restored `suspend`